test/ffmpeg-vaapi/encode/hevc.py

```diff
@@ -10,6 +10,7 @@
 
 spec      = load_test_spec("hevc", "encode", "8bit")
 spec_r2r  = load_test_spec("hevc", "encode", "8bit", "r2r")
+spec_level = load_test_spec("hevc", "encode", "8bit", "level")
```

Comment:
Please just use `load_test_spec("hevc", "encode", "8bit")` key

Reply:
The `gen_hevc_cbr_level_variants` generator will select the right cases.

Reply:
ok, thanks.

Reply:
@uartie  PR Updated. please review.